### ISSUE TITLE
undefined method `fetch' for nil:NilClass in spaceship through deliver

### ISSUE THREAD

**Issue by jwilliams-handy** [fastlane/fastlane#9373]
### New Issue Checklist

- [ x ] Updated fastlane to the latest version
- [ x ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

+-------------------------------------------------------------------+-------------------------------------------------------------------+
|                                                        deliver 2.36.0 Summary                                                         |
+-------------------------------------------------------------------+-------------------------------------------------------------------+
| ipa                                                               | /Users/me/Developer/Company/iosv3/ExternalBuilds/CompanyConsumer  |
|                                                                   | -7.21.1.2.ipa                                                     |
| app_version                                                       | 7.21.1                                                            |
| app_identifier                                                    | com.company.company                                               |
| username                                                          | ios@company.com                                                   |
| submit_for_review                                                 | true                                                              |
| automatic_release                                                 | false                                                             |
| skip_screenshots                                                  | true                                                              |
| force                                                             | true                                                              |
| submission_information.add_id_info_limits_tracking                | true                                                              |
| submission_information.add_id_info_serves_ads                     | false                                                             |
| submission_information.add_id_info_tracks_action                  | true                                                              |
| submission_information.add_id_info_tracks_install                 | true                                                              |
| submission_information.add_id_info_uses_idfa                      | true                                                              |
| submission_information.content_rights_has_rights                  | true                                                              |
| submission_information.content_rights_contains_third_party_conte  | true                                                              |
| nt                                                                |                                                                   |
| submission_information.export_compliance_platform                 | ios                                                               |
| submission_information.export_compliance_compliance_required      | false                                                             |
| submission_information.export_compliance_encryption_updated       | false                                                             |
| submission_information.export_compliance_uses_encryption          | false                                                             |
| submission_information.export_compliance_is_exempt                | false                                                             |
| submission_information.export_compliance_contains_third_party_cr  | false                                                             |
| yptography                                                        |                                                                   |
| submission_information.export_compliance_contains_proprietary_cr  | false                                                             |
| yptography                                                        |                                                                   |
| submission_information.export_compliance_available_on_french_sto  | false                                                             |
| re                                                                |                                                                   |
| screenshots_path                                                  | ./fastlane/screenshots                                            |
| metadata_path                                                     | ./fastlane/metadata                                               |
| edit_live                                                         | false                                                             |
| platform                                                          | ios                                                               |
| skip_binary_upload                                                | false                                                             |
| skip_metadata                                                     | false                                                             |
| overwrite_screenshots                                             | false                                                             |
+-------------------------------------------------------------------+-------------------------------------------------------------------+

[19:05:30]: Making sure the latest version on iTunes Connect matches '7.21.1' from the ipa file...
[19:05:33]: --------------------------
[19:05:33]: --- Step: notification ---
[19:05:33]: --------------------------
+------------------------+-------------------------------------------------------------------+
|                                        Lane Context                                        |
+------------------------+-------------------------------------------------------------------+
| DEFAULT_PLATFORM       | ios                                                               |
| PLATFORM_NAME          | ios                                                               |
| LANE_NAME              | ios release                                                       |
| BUILD_NUMBER           | 2                                                                 |
| VERSION_NUMBER         | 7.21.1                                                            |
| IPA_OUTPUT_PATH        | /Users/me/Developer/Company/iosv3/ExternalBuilds/CompanyConsumer  |
|                        | -7.21.1.2.ipa                                                     |
| DSYM_OUTPUT_PATH       | /Users/me/Developer/Company/iosv3/ExternalBuilds/CompanyConsumer  |
|                        | -7.21.1.2.app.dSYM.zip                                            |
| XCODEBUILD_ARCHIVE     | /Users/me/Library/Developer/Xcode/Archives/2017-06-05/CompanyC  |
|                        | onsumer-7.21.1.2 2017-06-05 18.50.19.xcarchive                    |
| S3_IPA_OUTPUT_PATH     | https://mobile-ios-assets.s3.amazonaws.com/ConsumerVersions/Hand  |
|                        | yConsumer-7.21.1.2.ipa                                            |
| S3_DSYM_OUTPUT_PATH    | https://mobile-ios-assets.s3.amazonaws.com/ConsumerVersions/Hand  |
|                        | yConsumer-7.21.1.2.app.dSYM.zip                                   |
| S3_PLIST_OUTPUT_PATH   | https://mobile-ios-assets.s3.amazonaws.com/ConsumerVersions/Hand  |
|                        | yConsumer-7.21.1.2.plist                                          |
| S3_HTML_OUTPUT_PATH    | https://mobile-ios-assets.s3.amazonaws.com/Companyconsumer.html     |
| S3_VERSION_OUTPUT_PATH | https://mobile-ios-assets.s3.amazonaws.com/ConsumerVersions/curr  |
|                        | ent_consumer_version.json                                         |
+------------------------+-------------------------------------------------------------------+
[19:05:33]: undefined method `fetch' for nil:NilClass

+------+------------------------------------------+-------------+
|                       fastlane summary                        |
+------+------------------------------------------+-------------+
| Step | Action                                   | Time (in s) |
+------+------------------------------------------+-------------+
| 1    | Verifying required fastlane version      | 0           |
| 2    | default_platform                         | 0           |
| 3    | Switch to ios                            | 0           |
|      | ensure_git_branch_clean_and_latest lane  |             |
| 4    | git_pull                                 | 0           |
| 5    | Switch to ios update_version_number lane | 0           |
| 6    | Switch to ios                            | 0           |
|      | ensure_git_branch_clean_and_latest lane  |             |
| 7    | git_pull                                 | 0           |
| 8    | Switch to ios bump_build_number lane     | 0           |
| 9    | increment_build_number                   | 3           |
| 10   | get_version_number                       | 0           |
| 11   | get_build_number                         | 0           |
| 12   | commit_version_bump                      | 1           |
| 13   | git tag -l | xargs git tag -d            | 0           |
| 14   | git fetch --tags                         | 3           |
| 15   | git_tag_exists                           | 0           |
| 16   | add_git_tag                              | 0           |
| 17   | push_to_git_remote                       | 2           |
| 18   | gym                                      | 224         |
| 19   | crashlytics                              | 197         |
| 20   | upload_symbols_to_crashlytics            | 30          |
| 21   | s3                                       | 465         |
| 💥   | deliver                                  | 5           |
| 23   | notification                             | 0           |
+------+------------------------------------------+-------------+

[19:05:33]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️  undefined method `edit_version' for nil:NilClass
    https://github.com/fastlane/fastlane/issues/8660 [closed] 8 💬
    a week ago

➡️  Undefined method `fetch` for nil:NilClass (NoMethodError) again?
    https://github.com/fastlane/fastlane/issues/7307 [closed] 4 💬
    3 weeks ago

➡️  [!] undefined method `native_targets' for #<Xcodeproj::Project:[MEMORY_INSTANCE]>
    https://github.com/fastlane/fastlane/issues/8935 [closed] 2 💬
    6 weeks ago

and 62 more at: https://github.com/fastlane/fastlane/search?q=undefined%20method%20%60fetch'%20for%20nil&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (/Users/me/.rbenv/versions/2.1.8/bin/fastlane)
NoMethodError: [!] undefined method `fetch' for nil:NilClass
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:814:in `block in setup_trailers_for'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:813:in `each'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:813:in `setup_trailers_for'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:800:in `block in setup_trailers'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:798:in `each'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:798:in `setup_trailers'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:381:in `setup'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/base.rb:215:in `initialize'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:226:in `new'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:226:in `factory'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/app_version.rb:247:in `find'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/application.rb:111:in `edit_version'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/spaceship/lib/spaceship/tunes/application.rb:212:in `ensure_version!'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/deliver/lib/deliver/runner.rb:39:in `verify_version'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/deliver/lib/deliver/runner.rb:20:in `run'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/actions/deliver.rb:16:in `run'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:226:in `chdir'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
  Fastfile:176:in `block (2 levels) in parsing_binding'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:153:in `run'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/commander-fastlane-4.4.4/lib/commander/runner.rb:476:in `run_active_command'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:39:in `run!'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/commander-fastlane-4.4.4/lib/commander/delegates.rb:15:in `run!'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
  /Users/me/.rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/fastlane-2.36.0/bin/fastlane:20:in `<top (required)>'
  /Users/me/.rbenv/versions/2.1.8/bin/fastlane:23:in `load'
  /Users/me/.rbenv/versions/2.1.8/bin/fastlane:23:in `<top (required)>'

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.5                                     |
| Ruby                        | 2.1.8                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.6.1                           |
| Installation Source         | ~/.rbenv/versions/2.1.8/bin/fastlane        |
| Host                        | Mac OS X 10.12.5 (16F73)                    |
| Ruby Lib Dir                | ~/.rbenv/versions/2.1.8/lib                 |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.2                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.19.3"

default_platform :ios

platform :ios do
  before_all do
    ENV["SLACK_URL"] = "..."    
  end

  lane :unit_tests do
    scan(
      devices: ["iPhone 6s (9.3)", "iPhone 7 (10.2)"],
      scheme: "Company Internal",
      skip_slack: true
      )
  end

  lane :core_tests do
    scan(
      device: "iPhone 6s (9.1)",
      scheme: "Company TestAutomation",
      skip_slack: true
      )
  end

  lane :beta_environment do |options|
    raise "Must provide a valid environment" if options[:env].nil? || options[:env].empty?

    environment = options[:env]

    version_hash = update_version_number(
      version_bump_type: options[:version_bump_type], 
      branch_override: options[:branch_override]
    )

    final_version_number = version_hash[:final_version_number]

    xcargs = "GCC_PREPROCESSOR_DEFINITIONS='${inherited} ENVIRONMENT_STR=#{environment}'"

    build_directory = "../EnvironmentBuilds"
    file_prefix = "CompanyConsumer-#{final_version_number}-#{environment}"

    gym(
      toolchain: "swift_3_0",
      export_method: "enterprise",
      clean: true,
      scheme: "Company Internal",
      configuration: "DistributionInternal",
      xcargs: xcargs,
      output_directory: build_directory,
      output_name: "#{file_prefix}.ipa"
    )

    crashlytics_api_token = ENV["CRASHLYTICS_API_TOKEN"]

    crashlytics(
      crashlytics_path: "Pods/Crashlytics",

      ipa_path: "#{build_directory}/#{file_prefix}.ipa",
      api_token: crashlytics_api_token,
      build_secret: ENV["CRASHLYTICS_BUILD_SECRET"],
    )

    upload_symbols_to_crashlytics(
      dsym_path: "#{build_directory}/#{file_prefix}.app.dSYM.zip",
      api_token: crashlytics_api_token
    )
  end

  lane :beta do |options|
    version_hash = update_version_number(
      version_bump_type: options[:version_bump_type], 
      branch_override: options[:branch_override]
    )

    final_version_number = version_hash[:final_version_number]

    build_directory = "../InternalBuilds"
    file_prefix = "CompanyConsumer-#{final_version_number}"

    gym(
      toolchain: "swift_3_0",
      export_method: "enterprise",
      clean: true,
      scheme: "Company Internal",
      configuration: "DistributionInternal",
      output_directory: build_directory,
      output_name: "#{file_prefix}.ipa"
    )

    crashlytics_api_token = ENV["CRASHLYTICS_API_TOKEN"]

    crashlytics(
      crashlytics_path: "Pods/Crashlytics",
      ipa_path: "#{build_directory}/#{file_prefix}.ipa",
      api_token: crashlytics_api_token,
      build_secret: ENV["CRASHLYTICS_BUILD_SECRET"],
      groups: ["ios-consumer-beta-testers"]
    )

    upload_symbols_to_crashlytics(
      dsym_path: "#{build_directory}/#{file_prefix}.app.dSYM.zip",
      api_token: crashlytics_api_token
    )

    slack(
      message: "New internal iOS build uploaded to Crashlytics! Get it while it's hot 🔥!",
      channel: "consumer-app-releases"
    )
  end

  lane :release do |options|
    ensure_git_branch_clean_and_latest(branch_override: options[:branch_override])   

    version_hash = update_version_number(
      version_bump_type: options[:version_bump_type], 
      branch_override: options[:branch_override]
    )

    final_version_number = version_hash[:final_version_number]

    build_directory = "../ExternalBuilds"
    file_prefix = "CompanyConsumer-#{final_version_number}"

    gym(
      toolchain: "swift_3_0",
      export_method: "app-store",
      clean: true,
      scheme: "Company External",
      configuration: "DistributionExternal",
      output_directory: build_directory,
      output_name: "#{file_prefix}.ipa"
    )

    ipa_path = "#{build_directory}/#{file_prefix}.ipa"
    dsym_path = "#{build_directory}/#{file_prefix}.app.dSYM.zip"
    crashlytics_api_token = ENV["CRASHLYTICS_API_TOKEN"]

    crashlytics(
      crashlytics_path: "Pods/Crashlytics",
      ipa_path: ipa_path,
      api_token: crashlytics_api_token,
      build_secret: ENV["CRASHLYTICS_BUILD_SECRET"],
    )

    upload_symbols_to_crashlytics(
      dsym_path: dsym_path,
      api_token: crashlytics_api_token
    )

    s3(
      ipa: ipa_path,
      dsym: dsym_path,
      bucket: "mobile-ios-assets",
      path: "ConsumerVersions/",
      upload_metadata: "true",
      plist_template_path: "fastlane/company_plist_template",
      plist_file_name: "ConsumerVersions/#{file_prefix}.plist",
      version_file_name: "ConsumerVersions/current_consumer_version.json",
      html_file_name: "companyconsumer.html"
    )

    deliver(
      ipa: ipa_path,
      app_version: version_hash[:version_number],
    )

    slack(
      message: "New iOS build submitted to the App Store! Waiting on 👍 from 🍏",
      channel: "consumer-app-releases"
    )
  end

  lane :pull_request do |options|
    error_message = "Please provide a title for your pull request i.e. title:'My title'"

    title = options[:title]
    if title.nil? || title.empty?
      UI.error(error_message)
      raise error_message
    end

    pull_request_url = create_pull_request(
      api_token: ENV['GITHUB_TOKEN'],
      repo: 'Companybook/ios-consumer',
      title: title,
      base: 'master'
    )

    slack(
      message: "Looking for 👀 on a new PR: #{title} #{pull_request_url}",
      channel: "eng-ios"
    )
  end

  lane :update_version_number do |options|
    ensure_git_branch_clean_and_latest(branch_override: options[:branch_override]) 
    version_bump_type = options[:version_bump_type]

    valid_bump_types = ["major", "minor", "patch", "build", "skip"]
    
    valid = false
    valid_bump_types.each do |type|
      valid = type == version_bump_type
      break if valid
    end

    error_message = "Must pass a version_bump_type value i.e. fastlane [lane] version_bump_type:minor"
    if !valid
      UI.error(error_message)
      raise error_message
    end

    requires_git_push = false
    
    if version_bump_type != "skip"
      if version_bump_type == "build"
        bump_build_number()
      else
        bump_version(bump_type: version_bump_type)
      end
      requires_git_push = true 
    end

    version_number = get_version_number()
    build_number = get_build_number()
    final_version_number = version_number + "." + build_number

    commit_version_bump(message: "Version bump to " + final_version_number) if requires_git_push    

    # Remove all tags not on remote
    sh("git tag -l | xargs git tag -d")
    sh("git fetch --tags")

    if !git_tag_exists(tag: final_version_number)
      add_git_tag(tag: final_version_number)
      requires_git_push = true
    end

    push_to_git_remote if requires_git_push

    { final_version_number: final_version_number, version_number: version_number, build_number: build_number }
  end

  private_lane :ensure_git_branch_clean_and_latest do |options|
    git_pull
    unless options[:branch_override]
      ensure_git_status_clean
      ensure_git_branch(branch: 'master') 
    end
  end

  private_lane :bump_version do |options|
    increment_version_number(
      bump_type: options[:bump_type]
    )
    increment_build_number(
      build_number: 1
    )
  end

  private_lane :bump_build_number do
    version = increment_build_number()
  end

  # You can define as many lanes as you want

  after_all do |lane|
    notification(
      title: "Fastlane #{lane}",
      message: "Lane completed"
    ) 
  end

  error do |lane, exception|
    notification(
      title: "Fastlane #{lane} error",
      message: "#{exception.message}"
    )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
apple_id "ios@company.com"


for_lane :external do
  app_identifier "com.companybook.Company"
  team_id "B3348384G6"
end

for_lane :internal do
  app_identifier "com.company.internal"
  team_id "M77QCHTG8V"
end

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md

```
</details>

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.36.0  | 🚫 Update available |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| bundler                   | 1.14.4       |
| CFPropertyList            | 2.3.5        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| json                      | 1.8.6        |
| mini_portile2             | 2.1.0        |
| nokogiri                  | 1.7.0.1      |
| aws-sdk-v1                | 1.66.0       |
| aws-sdk                   | 1.66.0       |
| babosa                    | 1.0.2        |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.4        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.56.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.12.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.11.0.1     |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| hurley                    | 0.2          |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| google-api-client         | 0.9.28       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.2.1        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.5.0        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.4        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 0.0.4        |
</details>

*generated on:* **2017-06-05**
</details>

**Comment by burgalon** [comment]
I think there has been some change in iTunes after WWDC :)

**Comment by tsafrir** [comment]
Check latest release. This one should be fixed by https://github.com/fastlane/fastlane/issues/9367

**Comment by jwilliams-handy** [comment]
Thanks @tsafrir

**Comment by sungjin0213** [comment]
As @tsafrir said, for me, this was fixed with latest release (fastlane 2.37.0)
